NAMESPACE

```diff
@@ -1,5 +1,8 @@
 # Generated by roxygen2: do not edit by hand
 
+export(accelerometer_features)
+export(getHR)
```

Comment:
We'll merge Megha's PR first. Change this to `get_heartrate` and we can pull this one in afterwards.